src/Samples/Sample.AspNetCore/Views/Verify/ViewList.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿@model SwedbankPay.Sdk.Payments.CardPayments.CardPayment
+@{
+    ViewData["Title"] = "ViewList";
+}
+
+<h1>ViewList</h1>
+
+<pre id="swedbank-pay-seamless-verificationList">
+    Here is the verificationList: @Html.Raw(Newtonsoft.Json.JsonConvert.SerializeObject(Model.PaymentResponse.Verifications.VerificationList, Newtonsoft.Json.Formatting.Indented))
+</pre>
```

Comment:
```suggestion
<div> Here is the verificationList: 
<pre id="swedbank-pay-seamless-verificationList">
@Html.Raw(Newtonsoft.Json.JsonConvert.SerializeObject(Model.PaymentResponse.Verifications.VerificationList, Newtonsoft.Json.Formatting.Indented))
</pre></div>
```
Concider moving the the text part out to its own element. This will make it easier to test the content of the `<pre>` later.